homeassistant/components/suez_water/sensor.py

```diff
@@ -0,0 +1,118 @@
+"""Sensor for Suez Water Consumption data."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, VOLUME_LITERS
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+CONF_COUNTER_ID = 'counter_id'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=720)
```

Comment:
Would be easier to read if we use hours.

```suggestion
MIN_TIME_BETWEEN_UPDATES = timedelta(hours=12)
```